src/components/MarketMySharesForm/index.js

```diff
@@ -93,18 +104,93 @@ class MarketMySharesForm extends Component {
     }
 
     if (decimalValue.lt(0)) {
-      return 'Number can\'t be negative.'
+      return "Number can't be negative."
     }
 
-    if (decimalValue.gt(Decimal(props.marketShares.filter(
-      share => share.id === this.state.extendedSellId)[0].balance,
-    ).div(1e18).toString())) {
-      return 'You\'re trying to sell more than you invested.'
+    if (
```

Comment:
Refactor to something like:
```
const myUsefulDecimalName = Decimal(props.marketShares.filter(share => share.id ....).div(1e18).toString(),
if (decimalValue.gt(myUsefulDecimalName)) { 
```

Reply:
@apanizo I completely agree that it should be refactored, but i haven't changed those components. I've only fixed eslint errors and formatting in those files.